.github/workflows/skyportal-fink-client.yml

```diff
@@ -0,0 +1,161 @@
+        test_subset: ['Unit Testing of utils functions (with skyportal API)','Test Polling Alerts']
+
+    steps:
+      - uses: actions/setup-python@v1
+        with:
+          python-version: "3.8"
+
+      - name: Checkout repository and submodules
+        uses: actions/checkout@v2
+        with:
+          submodules: recursive
+
+      - uses: actions/cache@v2
+        with:
+          path: |
+            ~/.npm
+          key: ${{ runner.os }}-npm-${{ hashFiles('package.json') }}
+
+      - uses: actions/cache@v2
+        with:
+          path: |
+            ~/.cache/pip
+          key: ${{ runner.os }}-${{ hashFiles('**/requirements*.txt') }}
+
+      - name: Write SkyPortal configs
+        working-directory: ./skyportal
+        run: |
+          cat << EOF > config.yaml
+            database:
+              database: skyportal
+              host: localhost
+              port: 5432
+              user: skyportal
+              password: anything
+          EOF
+          cat << EOF > test_config.yaml
+            database:
+              database: skyportal_test
+              user: skyportal
+              host: localhost
+              port: 5432
+              password: anything
+            server:
+              auth:
+                debug_login: True
+          EOF
+      - name: Install system dependencies
+        working-directory: ./skyportal
+        run: |
+          curl -sL https://deb.nodesource.com/setup_17.x | sudo -E bash -
+          sudo apt install -y wget nodejs unzip firefox nginx libcurl4-gnutls-dev libgnutls28-dev
+          pip install pip==21.3.1
+          pip install wheel numpy
+          export NPM_PACKAGES="${HOME}/.npm-packages"
+          export PATH=${NPM_PACKAGES}/bin:$PATH
+          export NODE_PATH="$NPM_PACKAGES/lib/node_modules:$NODE_PATH"
+          sudo npm -g install npm@latest
+          which python; python --version
+          echo npm $(npm --version)
+          echo node $(node --version)
+          nginx -v
+          firefox --version
+
+      - name: Initialize SkyPortal
+        working-directory: ./skyportal
+        run: |
+          # Usually, we create databases on the local machine, so
+          # `createdb` just works out of the box.  However, when doing
+          # this on a server, as in our case, we need to specify extra
+          # options.
+          #
+          # db_init should not complain if the databases exist already
+          #
+          echo "localhost:5432:*:skyportal:anything" > ~/.pgpass
+          chmod 600 ~/.pgpass
+          createdb -h localhost -U skyportal skyportal_test
+          psql -U skyportal -h localhost -c "GRANT ALL PRIVILEGES ON DATABASE skyportal_test TO skyportal;" skyportal_test
+          export NPM_CONFIG_LEGACY_PEER_DEPS="true"
+          make db_init
+          pip list --format=columns
+          npm ls --depth 0 || true
+          # Stamp Alembic HEAD
+          PYTHONPATH=. alembic -x config=config.yaml stamp head
```

Comment:
Why do you need to specify `PYTHONPATH=.` here?

Reply:
Same as above, we followed the template of SkyPortal's GA. I'll try removing it.

Reply:
@JulienPeloton  EDIT: Removing it causes the following error when initializing SkyPortal in the GA: 
`File "alembic/env.py", line 5, in <module>
    from baselayer.app.config import load_config
ModuleNotFoundError: No module named 'baselayer'`

Reply:
OK, we will see later why the pythonpath does not contain it